docs/pages/academy/1-GRAKN-intro/2-setup.md

```diff
@@ -20,33 +20,22 @@ If you do not know where to begin with, try having look at this [tutorial](http:
 You will also need to have and know how to use a text editor which can save simple, non formatted text files. There are countless alternatives on every platform, just pick your favourite one.
 
 
-## Setup the Academy Virtual Machine (VM)
-
-To make sure that the GRAKN environment is the same for everyone following this course, we will use a virtual machine (VM from now on) with GRAKN pre-installed and the data we will be using pre-loaded.
-
-To set it up you have to follow some simple steps:
-
-  1. Install Oracle’s VirtualBox if you have not already installed it. It can be downloaded from [here](https://www.virtualbox.org/wiki/Downloads) where you will also find detailed instructions on how to set it up on your favourite OS (it is a fairly simple process).
-
-  1. Download the GRAKN virtual machine following [this link](http://grakn.ai/download-academy) and unzip the downloaded file
-
-  1. Start Virtualbox and from the `File` menu select `Import Appliance`. Open the `GRAKN.ovf` file that you will find in the folder you unzipped in the preceding step. This will import the virtual machine image into VirtualBox
-
-  1. You are ready to go: start the Virtual Machine you just imported and you will be presented a login screen to the virtual machine. Input `grakn` both as a user name and password and you will be in the shell.
-
-![VM Screenshot](/images/academy/1-welcome/VM-screenshot.png)
-
-### Exercise
-
-Once you have started the virtual machine, list the content on the home directory, then move yourself into the folder `grakn/` from here you will be able to start grakn.
+## Setup Grakn
 
+You can download the latest version of Grakn core from [here](https://grakn.ai/download/), unzip it, put it in a folder you like. 
+Or you can follow the instruction on [this page](https://dev.grakn.ai/docs/get-started/setup-guide).
 
 ## Starting and using GRAKN
 
-With the Virtual Machine up and running, you are ready to start grakn. From the directory `~/grakn/` run the command `./grakn server start` to start Grakn. As soon as you see the Grakn logo, you will be able to connect to the Grakn.
+From the directory of grakn, run the command `./grakn server start` to start Grakn. As soon as you see the Grakn logo, you will be able to connect to the Grakn.
```

Comment:
- "... directory of grakn, run..."
- "...start Grakn. As soon..."
- "...as you see the Grakn logo, you..."

Let's spell it "GRAKN" to be consistent with the rest of the docs